core/db/migrate/20150609093816_increase_scale_on_pre_tax_amounts.rb

```diff
@@ -0,0 +1,16 @@
+class IncreaseScaleOnPreTaxAmounts < ActiveRecord::Migration
+  def change
+    # set pre_tax_amount on shipments to discounted_amount - included_tax_total
+    # so that the null: false option on the shipment pre_tax_amount doesn't generate
+    # errors.
+    #
```

Comment:
can we do this with a 
```ruby
Spree::Shipment.where(pre_tax_amount: nil).update_all('pre_tax_amount = (cost + promo_total) - included_tax_total)
```
rather than the exec? its a nit-pick but will ensure right table names or whatever are used.